samples/packages/pulumi-typescript/tests/test-bootstrap.ts

```diff
@@ -0,0 +1,99 @@
+// this is the code that would be in a project bootstrap file. It pulls the stack together using a config and CDF ochestrator
+import * as cdf from '@openfabr/cdf'
+import { PackageInfraConfig, PackageInfraPlanConstructs, PackagePlanner } from '../src';
+import config from './short-config.json'
+import * as aws from '@pulumi/aws';
+import { InfraPlanConstructs, GeneralConfig, NetworkConfig, ComponentConfig, ServiceConfig, RelationConfig, InfraConfig, Planner, Custom, ResultHandler, Orchestrator, InfraPlanOutputs } from '@openfabr/cdf';
+import { ProjectComponent, ProjectComponentOpts } from '@openfabr/cdf/build/main/lib/bootstrap/pulumi/component';
+import * as pulumi from '@pulumi/pulumi';
+import { ComponentResource, Output } from '@pulumi/pulumi';
+import {error, log} from 'console'
+
+
+class LocalProjectComponent<
+  IPC extends InfraPlanConstructs,
+  GC extends GeneralConfig,
+  NC extends NetworkConfig,
+  CC extends ComponentConfig,
+  SC extends ServiceConfig,
+  RC extends RelationConfig
+> extends ComponentResource {
+  //readonly network: Network;
+
+  readonly outputs: Map<string, any>;
+
+  constructor(
+    name: string,
+    props: ProjectComponentOpts<IPC, GC, NC, CC, SC, RC>
+  ) {
+    super('fabr:cdf:Project', name, {}, props);
+
+    const result = props.orchestrator.runIn(this);
+
+    let o: InfraPlanOutputs;
+
+    new ResultHandler(
+      (outputs: cdf.InfraPlanOutputs) => {
+        o = outputs; //.get("network") as Network
```

Comment:
We need to be able to expose the `ComponentResource`s in a *.ts file like this to make them available to the Jest unit tests.

If this pattern is the best, then this and supporting code would move into `cdf/.../pulumi/component.ts`.